homeassistant/components/homekit/type_thermostats.py

```diff
@@ -623,8 +625,10 @@ def async_update_state(self, new_state: State) -> None:
                 )
 
         # Set current operation mode for supported thermostats
-        if hvac_action := attributes.get(ATTR_HVAC_ACTION):
-            homekit_hvac_action = HC_HASS_TO_HOMEKIT_ACTION[hvac_action]
+        if (hvac_action := attributes.get(ATTR_HVAC_ACTION)) and (
+            homekit_hvac_action := HC_HASS_TO_HOMEKIT_ACTION.get(hvac_action),
+            HC_HEAT_COOL_OFF,
+        ):
```

Comment:
Should `HC_HEAT_COOL_OFF` be placed in `get`?

Reply:
Good catch

Reply:
No, I already moved it in https://github.com/home-assistant/core/pull/123943